Marlin/Conditionals_LCD.h

```diff
@@ -92,7 +101,7 @@
     #endif
   #endif
 
-  #if ENABLED(REPRAP_DISCOUNT_FULL_GRAPHIC_SMART_CONTROLLER)
+  #if ENABLED(REPRAP_DISCOUNT_FULL_GRAPHIC_SMART_CONTROLLER) || ENABLED(ANET_FULL_GRAPHICS_LCD)
```

Comment:
That's what I wrote. You can completely remove all changes related to ANET_FULL_GRAPHICS_LCD if the pins_ANET_10.h is changed to only use REPRAP_DISCOUNT_FULL_GRAPHIC_SMART_CONTROLLER for Reprap Discount with wiring adaptor (and A6 pinout) and Anet A6 LCD (with A6 pinout). The second (instead of the third) option would be the A8 20x4 lcd which is really an unique piece of hardware.